go/README.md

```diff
@@ -0,0 +1,72 @@
+# Sqlcommenter [In development]
+
+SQLcommenter is a plugin/middleware/wrapper to augment application related information/tags with SQL Statements that can be used later to correlate user code with SQL statements.
+
+## Installation
+
+### Install from source
+
+* Clone the source
+* In terminal go inside the client folder location where we need to import google-sqlcommenter package and enter the below commands
+
+```shell
+go mod edit -replace google.com/sqlcommenter=path/to/google/sqlcommenter/go
+
+go mod tiny
+```
+### Install from github [To be added]
```

Comment:
Now, we have updated the mod file and also done the CI setup, should we update this section ?

Reply:
Shall we update once we figure out the package and path of publishing?